src/components/Playground/OAPlayground.vue

```
@@ -78,6 +89,95 @@
+  const innerResponse: PlaygroundResponse = {
+    body: null,
+    type: '',
+    time: null,
+    status: null,
+  }
+
+  trackEvent()
+
+  const start = performance.now()
+  const controller = new AbortController()
+  const timeoutId = setTimeout(() => controller.abort(), 30000) // 30s timeout
```

ESLint

[error] 'timeoutId' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)